src/main/java/de/codecrafter47/taboverlay/config/dsl/CustomPlaceholderConfiguration.java

```diff
@@ -292,16 +296,28 @@ public ColorAnimation() {
         @Override
         public PlaceholderBuilder<?, ?> bindArgs(PlaceholderBuilder<Context, ?> builder, List<PlaceholderArg> args, TemplateCreationContext tcc) {
             List<TextColor> colors = new ArrayList<>();
+            List<TextColor> formats = new ArrayList<>();
 
-            if (ConfigValidationUtil.checkNotNull(tcc, "!color_animation custom placeholder", "colors", this.colors, getStartMark())
-                    && ConfigValidationUtil.checkNotEmpty(tcc,  "!color_animation custom placeholder", "colors", this.colors, this.colors.getStartMark())) {
+            if (ConfigValidationUtil.checkNotNull(tcc,  "!color_animation custom placeholder", "colors", this.colors, this.colors.getStartMark()) 
```

Comment:
There are two spaces in front of `"!color_animation custom placeholder"`. You should use `getStartMark()` over `this.colors.getStartMark()`, as the latter might throw a null pointer exception. There's a space at the end of the line.

Reply:
Interesting point considering that `this.colors.getStartMark()` is from you here. I barely touched this part here.